Makefile

```diff
@@ -24,6 +27,14 @@ install_tools:
 lint:
 	scripts/lint.sh
 
+test: unit_test integration_test
+
+unit_test:
+	$(GOTEST) $$(go list ./... | grep -v $(INTEGRATION_TEST_PACKAGES))
```

Comment:
There may be another way to do this: you can use build tags to mark different kind of test packages!

Reply:
Build tags seems to have some cons in static analysis: errors, code completion, etc.

How about separate different kind of tests by folder or env variables?
https://peter.bourgon.org/blog/2021/04/02/dont-use-build-tags-for-integration-tests.html